spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,50 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] @paulinagonzalezc am getting the error below when running your tests using ` rspec spec `, this is caused by the  lack of finished setup of the helper methods the last screenshot below shows a pending statement that you need to add examples,(if you are not sure how to config them you can choose to remove them, since the tests run without them)
![image](https://user-images.githubusercontent.com/61408860/221972170-7422c9d3-8f12-4b3b-807e-0cc5eb57b251.png)
![image](https://user-images.githubusercontent.com/61408860/221972944-7da84476-7d60-4959-8f10-70b29066a08a.png)

Reply:
Thank you @danmainah , I will remove those tests 😄